backend/pricing/pricing.go

```diff
@@ -1313,8 +1313,11 @@ func (w *Worker) GetBillingIssue(ctx context.Context, workspace *model.Workspace
 	for _, paymentMethod := range i.PaymentMethodList().Data {
 		paymentMethods += 1
 		if paymentMethod.Card != nil && paymentMethod.Card.Checks != nil {
-			if paymentMethod.Card.Checks.CVCCheck == stripe.PaymentMethodCardChecksCVCCheckFail {
-				log.WithContext(ctx).WithField("customer", customer.ID).Info("stripe cvc check failed")
+			if paymentMethod.Card.Checks.CVCCheck == stripe.PaymentMethodCardChecksCVCCheckFail && paymentMethod.Card.Checks.AddressPostalCodeCheck == stripe.PaymentMethodCardChecksAddressPostalCodeCheckFail {
```

Comment:
Curious if this should be an `||` vs an `&&`?

Reply:
trying to make this stricter - only fail the card if the CVC check fails AND the ZIP code check fails. otherwise consider the payment method valid (since in the mastercard instance, the card is valid even tho the CVC check fails, but the zip check passes)